content/blog/2023/2022blog.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "2022 Progress!"
+author: "ghostway"
+published: 2023-01-14
+---
+
+2022 has been a great year for Leela. A lot of new contributors appeared and made significant improvements; overall, Leela has become considerably stronger and more interesting.
+<!--more-->
+
+While 2021 had [ceres](https://lczero.org/blog/2021/01/announcing-ceres/) written and small (but exciting) improvements to the training pipeline,
+in 2022 there were huge changes to how the search was conducted, network architecture, available backends and more.
+I'll try to detail a bit on some of them.
+
+#### Network Architecture
+In the AlphaZero paper, Google used Resnets to predict the value and policies of the position. In 2022, *Arcturai* after a lot of work
```

Comment:
* (technically DeepMind is not part of Google, but that's fine to ignore)
* It makes sense to expain who Arcturai is (something like "*Acturai*, an active Leela project contributor,").  (also other contributors when they are mentioned first time in the text).

Reply:
isn't it a little implied?

Reply:
Maybe it makes sense to add another sentence in the "preface", telling something along the lines of "In 2022, a number of new contributors, including Acturai, danielmonoroe, and Almaudoh, began actively participating."

Oh, I see it's already in the first sentence.. Maybe list nicknames there in parentheses?

Also feel free to ignore the suggestion.